css/style.css

```
@@ -1018,6 +1029,33 @@ h2 a {
   #doc-langs {
     font-size: 11px;
   }
+
+  .header-right {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  margin-right: -10px;

```

This block is missing indentation:

```suggestion
    display: flex;
    align-items: center;
    gap: 8px;
    margin-right: -10px;
```

---

That doesn't really matter much, we need to fix the styling issue at some point to make it uniform, you can check https://github.com/expressjs/expressjs.com/issues/1596.

---

Agreed. Still, it felt worth noting as the rest of the PR was internally consistent.

---

fixed, thanks @dpopp07